spec/features/formats/news_article_spec.rb

```diff
@@ -25,54 +19,16 @@ def when_i_choose_this_document_type
   end
 
   def and_i_fill_in_the_form_fields
+    base_path = Edition.last.document_type.path_prefix + "/a-great-title"
+    stub_publishing_api_has_lookups(base_path => Document.last.content_id)
     fill_in "revision[title]", with: "A great title"
     fill_in "revision[summary]", with: "A great summary"
```

Comment:
I think even this is unnecessary!

Reply:
It is necessary, as otherwise the `PUT /content/` request is never made (presumably because the base path is not associated with this document, so a new document is created rather than making an edit to an existing one).

Reply:
Ah, I meant the `summary`. Should be unnecessary - trying in the real app, for example.

Reply:
Ah yes! Removed 👍